File: Sem- 1/C/While Loop/prime.c
#include<stdio.h>
void main(){
	printf("Prime Number");
	int n,i=1,count=0;
	printf("\nEnter any number: ");
	scanf("%d",&n);
	while(i<=n){
		if(n%i==0){
			count+=1;
		}
		i++;
	}
	if(count==2){
		printf("Prime Number");
	}
	else{
		printf("Not a Prime Number");
	}
}

File: Sem- 3/Data Structure/Array/Prime.java
import java.util.Scanner;
public class Prime{
	public static void main(String[] args){
		Scanner sc=new Scanner(System.in);
		System.out.println("Prime Number Detector");
		System.out.print("Enter any number: ");
		int n=sc.nextInt();
		int count=0;
		if(n<0){
			System.out.println("Not Defined");
		}else if(n==0||n==1){
			System.out.println("Neither Prime nor Composite");
		}else{
			for(int i=2;i<n/2;i++){
				if(n%i==0){
					count++;
				}
			}
			if(count==0){
				System.out.println("Prime");
			}else{
				System.out.println("Not Prime");
			}
		}
	}
}